dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/WindowCallbackWrapper.kt

```diff
@@ -130,7 +130,7 @@ internal class WindowCallbackWrapper(
         } else {
             customTargetName
         }
-        GlobalRum.get().addUserAction(RumActionType.CUSTOM, targetName, emptyMap())
+        GlobalRum.get().addUserAction(RumActionType.TAP, targetName, emptyMap())
```

Comment:
This interaction is `handleBackEvent`, and there's a BACK type for actions, maybe we should use that instead as it more precise

Reply:
thanks! I didn't know `back` type exists in the RUM schema. I've updated my PR.